packages/url-state-provider/src/index.ts

```diff
@@ -3,42 +3,29 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import LZString from "lz-string"
-import superstate from "./superstate"
+/* eslint-disable */
+// @ts-nocheck
```

Comment:
We might need also to describe why we switch off the ts-checks and eslint, or?

Reply:
yes I agree

Reply:
It is mentioned in the PR description, but I can put it here as well.